tests/acceptance/features/webUIPrivateLinks/accessingPrivateLinks.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Access private link
+   As a user I want to access item which I copied private link for
```

Comment:
(minor) in general it's another user who wants to access a resource from a private link that was copied from some other user

Reply:
I'd want to implement also tests for private link for shared files where it will be different user.  But that will come in follow up PR. But I think it's nice to test it also for the same user so we see the file is the same.